libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx

```diff
@@ -311,8 +284,8 @@ export class SixItemPicker {
       isNextItemAllowed: () =>
         minLetter === undefined || this.getStringItemIndex(value) > this.getStringItemIndex(minLetter),
       getNextItem: () => this.getNextItemByOperation(subtract),
-      getRoundtripItem: () => this.max,
-      isRoundtripPossible: () => this.max === undefined && this.roundtrip,
+      getRoundtripItem: () => this.max as string,
```

Comment:
Hm, can `max` not also be a number? So wouldn't it be better to write `() => String(this.max)` if you always want to returna  string?

Reply:
It should not be a number in this code path, but I think it is a good idea to use `String(..)` anyway -> Changed all `... as string` to `String(...)`